### DISCUSSION TITLE
Network error: Can't find end tag corresponding to start tag

### DISCUSSION THREAD

**Discussion by cclub1** [Steffo99/greed#151] [Q&A] [answered]
I am getting this error before adding a note to a order.

**Network error while calling send_message(), retrying in 5 secs...
Full error: Can't parse entities: can't find end tag corresponding to start tag b**

It has to do something with this lines:

 # If the done button has been pressed...
            elif callback.data == "cart_done":
                # End the loop
                break
        # Create an inline keyboard with a single skip button
        cancel = telegram.InlineKeyboardMarkup([[telegram.InlineKeyboardButton(self.loc.get("menu_skip"),
                                                                               callback_data="cmd_cancel")]])
        # Ask if the user wants to add notes to the order
        self.bot.send_message(self.chat.id, self.loc.get("ask_order_notes"), reply_markup=cancel)
        # Wait for user input
        notes = self.__wait_for_regex(r"(.*)", cancellable=True)
        # Create a new Order
        order = db.Order(user=self.user,
                         creation_date=datetime.datetime.now(),
                         notes=notes if not isinstance(notes, CancelSignal) else "")

**Comment by Steffo99** [comment, marked as answer]
The string you are trying to send contains an unmatched `<b>` (bold) tag.

All tags should be closed with `</b>`, otherwise Telegram will refuse to send the message.

Check the `ask_order_notes` string in the string file of your language!

**Comment by cclub1** [comment]
You where right. I had one tag hidden in the text. thank you very much
